fe/fe-core/src/main/java/org/apache/doris/catalog/authorizer/RangerHiveAccessController.java

```diff
@@ -99,20 +112,20 @@ private boolean checkPrivilege(UserIdentity currentUser, HiveAccessType accessTy
         request.setResource(resource);
 
         RangerAccessResult result = hivePlugin.isAccessAllowed(request, auditHandler);
-        auditHandler.flushAudit();
 
         if (result == null) {
-            LOG.warn(String.format("Error getting authorizer result, please check your ranger config. Request: %s",
-                    request));
+            LOG.warn(String.format("Error getting authorizer result, please check your ranger config. Make sure "
+                    + "ranger policy engine is initialized. Request: %s", request));
```

Comment:
Maybe we should tell user how to check Ranger's log, eg, add `log4j.xml` in `fe/conf`, in our document.

Reply:
fixed